tests/v2/validation/provisioning/permutations/permutations.go

```diff
@@ -130,9 +130,8 @@ func RunTestPermutations(s *suite.Suite, testNamePrefix string, client *rancher.
 						require.NoError(s.T(), err)
 
 						provisioning.VerifyRKE1Cluster(s.T(), client, testClusterConfig, clusterObject)
-						etcdVersion, err := componentchecks.CheckETCDVersion(client, nodes, clusterObject.ID)
+						err = componentchecks.CheckETCDVersion(client, nodes, clusterObject.ID)
```

Comment:
so, there's no version checking at all now? What is this test doing if it doesn't check the version?

Reply:
logging the output isn't checking the output for correct version or anything though. We should still check if the version is valid. Maybe instead of checking if it is not empty, we should just find out what is expected and base the test on that.

Reply:
Let's talk offline about this, there's a bit confusion here I think.

Reply:
Fixed this, running new jobs offline.